test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -579,7 +579,7 @@ public void UnsupportedDatabaseThrows(SupportedLanguages lang)
         /// <summary>
         /// Tests that when a user function throws an exception we'll retry executing that function once the lease timeout expires
         /// </summary>
-        [RetryTheory]
+        [Theory]
```

Comment:
Are you going to bring this back?

Reply:
oh yeah I forgot to revert this, thanks!